vsts/vsts.yaml

```diff
@@ -571,13 +571,6 @@ jobs:
           # The support team already has a process to address this, so our pipeline can treat stale signatures as warnings (instead of treating it as an error).
           TreatStaleSignatureAs: Warning
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-autoapplicability.AutoApplicability@1
```

Comment:
##[warning]* This tool is no longer supported. Please remove this build task from your build definition. For more information, please see: https://aka.ms/gdn-v1-deprecation